android/app/src/androidTest/java/org/fossasia/openevent/APITest.java

```diff
@@ -25,13 +26,13 @@
     public void testSpeakerAPIResponse() throws Exception {
         APIClient client = new APIClient();
         final CountDownLatch latch = new CountDownLatch(1);
-        client.getOpenEventAPI().getSpeakers(Urls.EVENT_ID).enqueue(new Callback<SpeakerResponseList>() {
+        client.getOpenEventAPI().getSpeakers().enqueue(new Callback<List<Speaker>>() {
```

Comment:
Right now, I'm fetching the data from 
[https://github.com/fossasia/open-event-orga-server/tree/development/samples/ots16](https://github.com/fossasia/open-event-orga-server/tree/development/samples/ots16)

Once the API is up and running, we can switch to it, shouldn't be a tough task.